doc/source/catalog_generation.rst

```diff
@@ -0,0 +1,361 @@
+
+.. note:: As photometry is not performed on observations that utilized grisms, prisms, polarizers, ramp filters, or quad filters, these elements were omitted from the above list.
+
+.. table:: Table 4: Default concentration index threshold values
+
+    +---------------------+----------------------+----------------------+
+    | Instrument/Detector | :math:`{CI_{lower}}` | :math:`{CI_{upper}}` |
+    +=====================+======================+======================+
+    | ACS/HRC             | 0.9                  | 1.6                  |
+    +---------------------+----------------------+----------------------+
+    | ACS/SBC             | 0.15                 | 0.45                 |
+    +---------------------+----------------------+----------------------+
+    | ACS/WFC             | 0.9                  | 1.23                 |
+    +---------------------+----------------------+----------------------+
+    | WFC3/IR             | 0.25                 | 0.55                 |
+    +---------------------+----------------------+----------------------+
+    | WFC3/UVIS           | 0.75                 | 1.0                  |
+    +---------------------+----------------------+----------------------+
+
+2.3.2.2: Assignment of flag value 4 (Saturated Source)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+A flag value of 4 is assigned to sources that are saturated. The process of identifying saturated sources starts by
+first transforming the input image XY coordinates of all pixels flagged as saturated in the data quality arrays of each
+input flc/flt.fits images (the images drizzled together to produce the drizzle-combined filter image being used to
+measure photometry) from non-rectified, non-distortion-corrected coordinates to the rectified, distortion-corrected
+frame of reference of the filter-combined image. We then identify impacted sources by cross-matching this list of
+saturated pixel coordinates against the positions of sources in the newly created source catalog and assign flag values
+where necessary.
+
+2.3.2.3: Assignment of flag value 8 (faint detection limit)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+A flag value of 8 is assigned to sources whose signal to noise ratio is below a predefined value. We define sources as
+being above the faint object limit if the following is true:
+
+.. math::
+    \Delta ABmag_{outer} \leq  \frac{2.5}{snr \cdot log(10))}
+
+Where
+    * :math:`{\Delta ABmag_{outer}}` is the outer aperture AB magnitude uncertainty
+    * :math:`{snr}` is the signal to noise ratio, which is 1.5 for ACS/WFC and 5.0 for all other detectors.
+
+2.3.2.4: Assignment of flag value 32 (false detection: swarm around saturated source)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+The source identification routine has been shown to falsely identify sources in regions near bright or saturated
```

Comment:
to identify falsely

Reply:
Issue resolved in commit [203ed51](https://github.com/spacetelescope/drizzlepac/pull/760/commits/203ed519b49749f7ad519df95710174ab98faffe)